src/client/styles/pie.scss

```diff
@@ -16,24 +16,86 @@
  *  limitations under the License
  *
  */
-pie-item {
-  display: flex;
-}
+ 
+.pie-card {
+  border: 1px solid #ccc;
+  position: relative;
+  box-sizing: border-box;
+  margin: 12px;
+  display: inline-block;
 
-pie-img {
-  background: url(/pieimg/pie-svg) center center/cover no-repeat;
-  width: 300px;
-  height: 200px;
-  transition: background .2s;
-}
+  &.pie-card--small {
+    width: 220px;
+  }
 
-.pie-title {
-  min-width: 200px;
-  min-height: 28px;
-  background-position: 0 100%;
-  transition: background-position .2s;
-}
-.pie-description {
-  min-height: 200px;
-  min-width: 300px;
+  &.pie-card--mobile {
+    width: 320px;
+
+    .pie-card-top {
+      display: flex;
+
+      & > pie-img,
+      & > div {
+        width: 50%;
+      }
+
+      & > div {
+        padding: 12px;
+      }
+    }
+
+    .pie-title {
+      margin-top: 60px;
+    }
+  }
+
+  pie-img {
+    background: url('/pieimg/pie-svg') center center/cover no-repeat;
+    width: 200px;
+    height: 150px;
+    display: block;
+  }
+
+  .pie-card-content {
+    padding: 12px;
+    border-top: 1px solid #ccc;
+  }
+
+  .pie-card-actions {
+    border-top: 1px solid var(--primary-color);
+    display: flex;
+    justify-content: space-between;
+    font-size: .8em;
+  }
+
+  .pie-title {
+    text-transform: uppercase;
+    font-weight: 600;
+  }
+
+  .pie-tags {
+    li {
+      display: inline;
+    }
+
+    li::before {
+      background-color: green;
+    }
+  }
+
+  .fav {
```

Comment:
Is the `fav` a pie-specific thing? If so, I'd rename it to e.g. `pie-fav`. Otherwise, I'd move it out of this file.

Reply:
fav is not pie specific, but it can be positioned differently depending on the parent element and context. So this rules apply to a fav that happens to be inside the pie-card. I could make it a ".fav.pie-fav" class, but isn't it better to use inbuilt cascading of css here?

Reply:
Oh, I didn't notice the indenting! Yeah, this is inside `.pie-card`, so SASS will convert it to `.pie-card .fav`. LGTM.